predictions_FB.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from fbprophet import Prophet
+import yfinance as yf
+import math
+import os
+import sys
+import pandas as pd
+pd.options.mode.chained_assignment = None
```

Comment:
Does this need to be added at the root? Perhaps we could move this inside a function using this?